recipes/ogre/1.x/conanfile.py

```diff
@@ -0,0 +1,298 @@
+        "OGRE_BUILD_RENDERSYSTEM_GL": [True, False],
+        "OGRE_BUILD_RENDERSYSTEM_GL3PLUS": [True, False],
+        "OGRE_BUILD_RTSHADERSYSTEM_SHADERS": [True, False],
+        "OGRE_BUILD_SAMPLES": [True, False],
+        "OGRE_BUILD_TESTS": [True, False],
+        "OGRE_BUILD_TOOLS": [True, False],
+        "OGRE_CONFIG_ENABLE_QUAD_BUFFER_STEREO": [True, False],
+        "OGRE_CONFIG_FILESYSTEM_UNICODE": [True, False],
+        "OGRE_CONFIG_THREADS": "ANY",
+        "OGRE_CONFIG_THREAD_PROVIDER": "ANY",
+        "OGRE_ENABLE_PRECOMPILED_HEADERS": [True, False],
+        "OGRE_INSTALL_PDB": [True, False],
+        "OGRE_INSTALL_SAMPLES": [True, False],
+        "OGRE_INSTALL_TOOLS": [True, False],
+        "OGRE_INSTALL_VSPROPS": [True, False],
+        "OGRE_NODELESS_POSITIONING": [True, False],
+        "OGRE_PROFILING_REMOTERY_PATH": "ANY",
+        "OGRE_RESOURCEMANAGER_STRICT": "ANY",
+        "ogre_static": [True, False],
+        "ogre_set_double": [True, False],
+        "ogre_glsupport_use_egl": [True, False],
+        "OpenEXR_Half_LIBRARY": "ANY",
+        "OpenEXR_Half_LIBRARY_DEBUG": "ANY",
+        "OpenEXR_INCLUDE_DIR": "ANY",
+        "OpenEXR_Iex_LIBRARY": "ANY",
+        "OpenEXR_Iex_LIBRARY_DEBUG": "ANY",
+        "OpenEXR_IlmImf_LIBRARY": "ANY",
+        "OpenEXR_IlmImf_LIBRARY_DEBUG": "ANY",
+        "OpenEXR_IlmThread_LIBRARY": "ANY",
+        "OpenEXR_IlmThread_LIBRARY_DEBUG": "ANY",
+        "Qt5_DIR": "ANY",
+        "SDL2_DIR": "ANY",
+        "pugixml_DIR": "ANY"
+    }
+
+    # default options copied from https://github.com/StatelessStudio/ogre-conan/blob/master/conanfile.py
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "cp_bin_dir": "bin",
+        "cp_media_dir": "Media",
+        "disable_plugins": False,
+        "OGRE_ASSERT_MODE": 1,
+        "OGRE_BUILD_COMPONENT_BITES": True,
+        "OGRE_BUILD_COMPONENT_CSHARP": False,
+        "OGRE_BUILD_COMPONENT_HLMS": True,
+        "OGRE_BUILD_COMPONENT_JAVA": False,
+        "OGRE_BUILD_COMPONENT_MESHLODGENERATOR": True,
+        "OGRE_BUILD_COMPONENT_OVERLAY": True,
+        "OGRE_BUILD_COMPONENT_OVERLAY_IMGUI": True,
+        "OGRE_BUILD_COMPONENT_PAGING": True,
+        "OGRE_BUILD_COMPONENT_PROPERTY": True,
+        "OGRE_BUILD_COMPONENT_PYTHON": True,
+        "OGRE_BUILD_COMPONENT_RTSHADERSYSTEM": True,
+        "OGRE_BUILD_COMPONENT_TERRAIN": True,
+        "OGRE_BUILD_COMPONENT_VOLUME": True,
+        "OGRE_BUILD_DEPENDENCIES": False,
+        "OGRE_BUILD_PLUGIN_BSP": True,
+        "OGRE_BUILD_PLUGIN_DOT_SCENE": True,
+        "OGRE_BUILD_PLUGIN_OCTREE": True,
+        "OGRE_BUILD_PLUGIN_PCZ": True,
+        "OGRE_BUILD_PLUGIN_PFX": True,
+        "OGRE_BUILD_PLUGIN_STBI": True,
+        "OGRE_BUILD_RENDERSYSTEM_D3D11": True,
+        "OGRE_BUILD_RENDERSYSTEM_GL": True,
+        "OGRE_BUILD_RENDERSYSTEM_GL3PLUS": True,
+        "OGRE_BUILD_RTSHADERSYSTEM_SHADERS": True,
+        "OGRE_BUILD_SAMPLES": True,
```

Comment:
I suggest we first explore how we can fix the build with ` "OGRE_BUILD_SAMPLES": True` since this was [requested ](https://www.ogre3d.org/2015/07/08/ogre-progress-report-june-2015#more-2493)heavily in the past.

Reply:
Conan Center does not package examples neither tests. Keep the option, but set it to False. People still can build from source.

Reply:
done